SQL/time_log_system_tls.sql

```diff
@@ -42,7 +43,9 @@ CREATE TABLE `clients` (
   `client_postal_code` varchar(35) DEFAULT NULL,
   `client_country` varchar(100) DEFAULT NULL,
   `client_description` text,
-  `by_user` int(11) NOT NULL
+  `by_user` int(11) NOT NULL,
+  PRIMARY KEY (`ID`),
+  KEY `client_postal_code` (`client_postal_code`)
```

Comment:
Unnecessary.

Reply:
I've made it unnecessary, will be removed later today.